Tests/ImageTintingTests.swift

```diff
@@ -65,7 +67,7 @@ class ImageTintingTests: XCTestCase {
         XCTAssertNotNil(tintedResult)
 
         BONAssertNotEqualImages(untintedResult!, tintedResult!)
-    }
+    }   
```

Comment:
Extra whitespace. (If you `brew install swiftlint` 0.16.1 or higher, it will catch this for you. You can also enable trimming trailing whitespace in Xcode → Preferences → Text Editing → Automatically trim trailing whitespace.)